server/tournaments/index.ts

```diff
@@ -1377,56 +1603,66 @@ const tourCommands: {basic: TourCommands, creation: TourCommands, moderation: To
 			tournament.update();
 		},
 		resetname: 'clearname',
-		clearname(tournament, user) {
+		clearname(target, room, user) {
+			if (!room) return this.requiresRoom();
+			const tournament = room.getGame(Tournament);
+			if (!tournament) return this.errorReply(`There is no tournament running.`);
 			if (tournament.name === tournament.baseFormat) return this.errorReply("The tournament does not have a name.");
 			tournament.name = tournament.baseFormat;
 			this.room!.send(`|tournament|update|${JSON.stringify({format: tournament.name})}`);
 			this.privateModAction(`${user.name} cleared the tournament's name.`);
 			this.modlog('TOUR CLEARNAME');
 			tournament.update();
 		},
-	},
-	moderation: {
 		begin: 'start',
-		start(tournament, user) {
+		start(target, room, user) {
+			if (!room) return this.requiresRoom();
+			const tournament = room.getGame(Tournament);
+			if (!tournament) return this.errorReply(`There is no tournament running.`);
 			if (tournament.startTournament(this)) {
 				this.room!.sendMods(`(${user.name} started the tournament.)`);
 			}
 		},
 		dq: 'disqualify',
-		disqualify(tournament, user, params, cmd) {
-			if (params.length < 1) {
-				return this.sendReply(`Usage: ${cmd} <user>`);
+		disqualify(target, room, user, connection, cmd) {
+			if (!room) return this.requiresRoom();
+			const tournament = room.getGame(Tournament);
+			if (!tournament) return this.errorReply(`There is no tournament running.`);
+			if (!target) {
+				return this.sendReply(`Usage: /tour ${cmd} <user>`);
 			}
-			const targetUser = Users.get(params[0]);
+			const [userid, reason] = target.split(',').map(item => item.trim());
+			const targetUser = Users.get(userid);
 			const targetUserid = toID(targetUser);
```

Comment:
`targetUser` can be null here so if you want to support disqualifying nonexistent User objects, you might want a fallback for `targetUserid`